### ISSUE TITLE
Support rfc6570 uritemplates in click handler

### ISSUE THREAD

**Issue by dschulten** [Kong/insomnia#1764]
[Feature Request] – APIs can contain rfc6570 uritemplates, e.g. `http://example.com{?personId*}`. Building on https://github.com/getinsomnia/insomnia/issues/874, Insomnia should allow to provide uritemplate variables when the user clicks such a UriTemplate:

![image](https://user-images.githubusercontent.com/107194/68068181-f8832280-fd52-11e9-8c0e-4bd3d546d64e.png)

**Comment by dschulten** [comment]
Feature branch: https://github.com/dschulten/insomnia/tree/feature/follow-rfc6570-uritemplates

Some advice would help.
In uritemplates you can have [composite values](https://tools.ietf.org/html/rfc6570#section-2.4.2), e.g. lists or dictionaries. E.g. if `personId` is a list `[123,456]`, then the template `http://example.com{?personId*}` expands to `http://example.com?personId=123&personId=456`. The same works for objects like `{city: 'Newport Beach',state: 'CA'}`.
It might be useful to allow users to edit such composite values in the multiline editor. When multiline is enabled, a button **Click to edit** allows to open the editor instead of an input field.
However, I found that this button submits the modal *and* opens the editor at once. Is there a way to prevent the submit event and instead open the editor and later return to the **Execute Request** modal?

**Comment by gschier** [comment]
I like this! Nice work. 

At first glance (just by the screenshot), it doesn't seem like this should depend on #874. Ideally it should be treated as completely separate and should even be able to be merged into the existing system of opening links in the browser.

Feel free to submit a PR for this, ideally without the included changes in #874
